src/dlgTriggerEditor.cpp

```diff
@@ -70,11 +70,14 @@ const QString msgInfoAddTimer = "Check the manual for <a href='http://wiki.mudle
 const QString msgInfoAddButton = "To add a new button: <b>1.</b> Add a new group to define a new Button bar in case you don't have any."
                                  "<b>2.</b> Add new buttons to a button bar."
                                  "Check the manual for <a href='http://wiki.mudlet.org/w/Manual:Contents'>more information</a>.";
-
-const QString msgInfoAddKey = "To add a new key binding <b>1.</b> add a new key <b>2.</b> click on <u><b>grab key</b></u> and then press your key combination. <b><u>NOTE:</u></b> If you want to bind "
-                              "a key combination you must hold down the modifier keys (e.g. control, shift etc.) down before clicking on grab key. "
-                              "<b>3.</b> Define a command that is executed when the key is hit. <b>4. <u>Activate</u></b> the new key binding."
-                              "Check the manual for <a href='http://wiki.mudlet.org/w/Manual:Contents'>more information</a>.";
+const QString msgInfoAddKey = "<p>To add a new key binding:"
```

Comment:
![Selection_378](https://user-images.githubusercontent.com/110988/54486180-cc2e9a80-4884-11e9-8985-57a62ceff7ec.png)

does not fit

Reply:
This will be cured in a PR that I have nearing completion to get the `QSpitter` working properly on the right hand side of the editor which allows the message area to resize as needed.

Reply:
OK